### PR TITLE
Update setup readme

### PR DESCRIPTION
**Pull request by ed-iov**
Added some detailed information in the readme to do the first local setup, added additional regtest.js sample for hsm1 use only and a logback file as example to do the logs configuration.

### CODE REVIEW

**Comment by lucasvuotto** [comment]
pipeline: run

**Comment by lucasvuotto** [comment]
pipeline: run

File: README.md [hunk 1]
```diff
@@ -8,10 +8,55 @@ All private keys used in the library are for testing only and not used in any pr
...
+### bitcoind
+- download
+  Current pipeline in jenkins is running bitcoind 0.17, 0.18.1 works for local execution.
+  - Go to https://bitcoin.org/en/release/v0.18.1 in this page there is the link to the download sources
+  https://bitcoincore.org/bin/bitcoin-core-0.18.1/
+  Select the *.tar.gz file according to your OS (for MacOS sillicon [bitcoin-0.18.1-osx64.tar.gz](https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-osx64.tar.gz))
+  - Decompress and copy the path to the folder.
+  - run in terminal `sudo cp <path to bitcoin-core folder>/bin/bitcoin* /usr/local/bin`
+  - create a folder called bitcoindata in your workspace
+  - run to test the bitcoind installation standalone `bitcoind -deprecatedrpc=generate -addresstype=legacy -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -datadir=<path to bitcoindata> $@`
```
Comment (kelvinator07) [inline README.md:20]:
```bash
   curl -O https://bitcoin.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-osx64.tar.gz
   tar -zxf bitcoin-0.18.1-osx64.tar.gz
   sudo mkdir -p /usr/local/bin
   sudo cp bitcoin-0.18.1/bin/bitcoin* /usr/local/bin/.
   rm -rf bitcoin-0.18.1*
   As a validation, you can run *bitcoind -daemon*. Run *bitcoin-cli stop* afterwards.
```

File: README.md [hunk 2]
```diff
@@ -24,6 +69,11 @@ All private keys used in the library are for testing only and not used in any pr
   - Optionally specify `federate.hsmConfigs.keyPath` to your local keys, if you happen to be testing different keys or your keys are located in a different path.
 4. 
 	-  Run `npm test` to run testcases without interruption or `npm run test-fail-fast` to stop execution at first testcase failure.
+  - `regtest.js` has by default not using the following mapping:
+    - federate1: no hsm signer
+    - federate2 and federate3: hsm1
+    - federate4 and federate5: hsm2 (modify using the same logic that in federate2 and 3 for changing to hsm1 in case of hsm2 not working).
+
```
Comment (kelvinator07) [inline README.md:76]: We Should Update regtest.js To Use fed1 - keyfile, fed2 & fed3 - hsm1, fed4 & fed5 - hsm2

File: README.md (hunk 1 above)
Comment (ed-iov) [reply README.md:20]: Thank you, I'll adjust the instructions.

File: README.md (hunk 2 above)
Comment (kelvinator07) [reply README.md:76]: The pipeline.conf Is Configured As I Described Above, Hence Line 72 To 75 Can Be Removed

Comment (nathanieliov) [inline README.md:76]: Maybe this should be a sub-section like notes or something.

**Review by nathanieliov — APPROVED** [review]
LGTM

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**